pkg/cluster/cluster.go

```diff
@@ -91,7 +91,6 @@ type Cluster struct {
 	currentProcess      Process
 	processMu           sync.RWMutex // protects the current operation for reporting, no need to hold the master mutex
 	specMu              sync.RWMutex // protects the spec for reporting, no need to hold the master mutex
-	streamApplications  []string
```

Comment:
we don't need to remember stream apllicationIDs when we can list all stream CRDs belonging to a Postgres cluster with a K8s API call